src/tools/dev/scripts/bv_support/bv_vtkm.sh

```diff
@@ -96,43 +96,10 @@ function bv_vtkm_dry_run
 #
 #
 # *************************************************************************** #
-
-function apply_patch_1
-{
-   patch -p0 << \EOF
-diff -c ./vtkm/cont/arg/TransportTagTopologyFieldIn.h.orig ./vtkm/cont/arg/TransportTagTopologyFieldIn.h
-*** ./vtkm/cont/arg/TransportTagTopologyFieldIn.h.orig	Tue Dec  8 12:55:32 2020
---- ./vtkm/cont/arg/TransportTagTopologyFieldIn.h	Tue Dec  8 12:55:49 2020
-***************
-*** 90,96 ****
---- 90,98 ----
-    {
-      if (object.GetNumberOfValues() != detail::TopologyDomainSize(inputDomain, TopologyElementTag()))
-      {
-+ #if 0
-        throw vtkm::cont::ErrorBadValue("Input array to worklet invocation the wrong size.");
-+ #endif
-      }
-  
-      return object.PrepareForInput(Device(), token);
-EOF
-
-    if [[ $? != 0 ]] ; then
-      warn "vtkm patch 1 failed."
-      return 1
-    fi
-    return 0;
-}
-
 function apply_vtkm_patch
 {
     info "Patching VTKm . . ."
 
-    apply_patch_1
-    if [[ $? != 0 ]] ; then
-       return 1
-    fi
-
```

Comment:
I'm a little concerned about removing this patch. I can see that the line numbering in the file has changed, so this patch would need to be updated at a minimum. I recall it fixed a problem with rectilinear meshes, but unfortunately I didn't put any information in here why the patch is needed. Let's leave it removed and if it causes any problems we can add it back.

Reply:
@brugger1  Good point on this patch. I removed them since I knew the second one had been fixed in a vtk-m commit, but this one does look like it is still in the source code. I can try it with it added back in, that would be no problem, I probably shouldn't have removed them both.

Reply:
Were you going to add the patch back in?

Reply:
@brugger1 Yes, didn't get a chance to test it yesterday. It built fine for me, so it is added back in.